src/enemy.js

```diff
@@ -1,65 +1,32 @@
+/* eslint-disable class-methods-use-this */
 // enemy class
 class Enemy {
-    // private properties
-    _element = document.createElement('div');
-    _width = 50; // enemy size (still not sure about the right size)
-    _height = 50;
-    _x;
-    _y;
-    _speed;
-    _selected; // boolean
-    _question;
-    _answer;
-
-    constructor(x, y) {
-        this._x = x;
-        this._y = y;
+    constructor(xPos, yPos) {
+        this.element = document.createElement('div').classList.add('enemy');
```

Comment:
This sets element to undefined as classList add method does not return the element